template/static/css/_colors.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
same as this

Reply:
i think i didn't do a clean build, i will redo this.